sw/device/lib/testing/json/mem.c

```diff
@@ -0,0 +1,62 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#define UJSON_SERDE_IMPL 1
+#include "sw/device/lib/testing/json/mem.h"
+
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/testing/test_framework/ujson_ottf.h"
+
+#define MODULE_ID MAKE_MODULE_ID('j', 's', 'm')
+
+status_t ujcmd_mem_read32(ujson_t *uj) {
+  mem_read32_req_t op;
+  mem_read32_resp_t resp;
+  uintptr_t address;
+  TRY(ujson_deserialize_mem_read32_req_t(uj, &op));
+  if (op.address & (sizeof(uint32_t) - 1)) {
```

Comment:
```suggestion
  if ((op.address % sizeof(uint32_t)) != 0) {
```
I think it's easier to read to explicit document the intention of alignment check